sdk/appconfiguration/app-configuration/package.json

```
@@ -96,38 +85,26 @@
     "tslib": "^2.2.0"
   },
   "devDependencies": {
+    "@azure-tools/test-credential": "^1.0.0",
```

I believe this should be at least 2.0.0 for package migrating to ESM/ vitest
```suggestion
    "@azure-tools/test-credential": "^2.0.0",
```